config/index.js

```diff
@@ -55,6 +55,9 @@ module.exports = {
 			url: 'www.paypal.com',
 			environment: 'production'
 		},
+		googlePay: {
+			merchantId: 'f5swrg87wym8bx37'
```

Comment:
do we need to blank this out for sandbox?

Reply:
@mcstover according to the docs and my testing, this is fine. It is required for production, but won't hurt anything if it's there for sandbox